src/scene/container/container-mixins/effectsMixin.ts

```diff
@@ -9,7 +9,7 @@ import type { Effect } from '../Effect';
 
 export interface EffectsMixinConstructor
 {
-    mask?: number | Container;
+    mask?: number | Container | null | undefined;
     filters?: Filter | Filter[];
```

Comment:
While we are here, what about `filters`? Shouldn't that allow null/undefined as well?